low-level/framework/base/sspl_constants.py

```diff
@@ -151,8 +151,8 @@ class RaidDataConfig(Enum):
     MISMATCH_COUNT_FILE = "/md/mismatch_cnt"
     STATE_COMMAND_RESPONSE = 'idle'
     MISMATCH_COUNT_RESPONSE = '0'
-    RAID_RESULT_DIR = "/tmp"
-    RAID_RESULT_FILE_PATH = "/tmp/result_raid_health_file"
+    RAID_RESULT_DIR = f"{DATA_PATH}raid_integrity/"
```

Comment:
check for code to create this folder if it doesnt exist

Reply:
Done